introduction.md

```diff
@@ -1 +1,4 @@
-My name is Gusromero and my favorite Code School path is the JavaScript one
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?